pagopa/io-app
# chore: [IOCOM-1742,IOCOM-1756] Disabled notifications banner in profile/settings home

## Short description
addition of said banner, only shown when notification permissions are not granted to the app
![Screenshot 2024-10-08 at 15 58 15](https://github.com/user-attachments/assets/c370d8c2-7fde-4180-8d72-fb3e87efd90a)

## List of changes proposed in this pull request
- banner
- required I18n
- selector for correct banner order rendering
- current banner component has been extrapolated in a separate one, in order to declutter the main screen

## How to test
navigate to the profile screen, and check that:
- with notifications disabled, the red banner is shown
- with notifications enabled, the fiscalCode one is shown, unless previously closed
- if previously closed, no banner is shown
- the notification banner opens the system's settings for the app, in the correct "notifications" section.
